kedro/ipython/__init__.py

```diff
@@ -187,6 +192,20 @@ def _find_kedro_project(current_dir: Path) -> Any:  # pragma: no cover
     return None
 
 
+def _guess_run_environment() -> str:  # pragma: no cover
```

Comment:
Can we add a test for this helper function?

Reply:
@AhdraMeraliQB It would be great to have a way to test it, but I don't think we have a nice way to simulate the actual environment. 

We can fake the environment variable if we really want to have some test for this, though I think it doesn't add much value as the current function is a very simple switch case.